Files/Views/LayoutModes/GenericFileBrowser2.xaml

```diff
@@ -310,14 +311,14 @@
 
                             <Button
                                 Grid.Column="2"
+                                HorizontalContentAlignment="Left"
```

Comment:
It might be simpler to make this change in the ```HeaderButtonStyle``` instead of doing it in each button individually.